src/ui-test/extension.test.ts

```diff
@@ -78,6 +78,7 @@ interface RuntimeOutput {
 const RUNTIME_FOLDER = path.join(projectPath, 'src', 'ui-test', 'runtimes');
 const WSDL_FILE = path.join(projectPath, 'src', 'test', 'address.wsdl');
 const WSDL_URL = webServer.getWSDLURL();
+const MAVEN_CALL_TIMEOUT = 300000;
```

Comment:
Could you please change it to `const MAVEN_CALL_TIMEOUT =  process.env["WSDL2REST_MAVEN_CALL_TIMEOUT"] || 300000` so it can be modifed by environment variables.